v1/pipeline/steps/cd/k8s-rolling-step-info.yaml

```diff
@@ -3,56 +3,20 @@ allOf:
 - $ref: ../../common/step-spec-type.yaml
 - type: object
   properties:
-    commandFlags:
+    command_flags:
```

Comment:
can we just name it flags?
and add description for this field?

Reply:
Yes, Done